include/golioth/client.h

```diff
@@ -63,6 +63,8 @@ enum golioth_auth_type {
     GOLIOTH_TLS_AUTH_TYPE_PSK,
     /// Authenticate with PKI certificates (CA cert, public client cert, private client key)
     GOLIOTH_TLS_AUTH_TYPE_PKI,
+    /// Authenticate with TLS credential tag (Zephyr specific)
```

Comment:
Nit in commit message: `%s/numeours/numerous/g`

Reply:
Fixed!